Always_green_CI.md

```diff
@@ -0,0 +1,56 @@
+# Introduction
+
+We have occasionally seen that the CI fails due to flaky tests or jobs, environment issues and more. This presents a challenge for new contributors as they need to spend time figuring out issues that are not related to their changes in the code. It also hampers the confidence the community has in the CI as a simple yes/no answer and instead talks about must vs optional tests. We believe that this will become an increasing bottleneck if the kata community is successful in kicking off the new confidential containers project which will bring in additional participating companies and individual contributors. Given this, we established a process and policy that should allow the project to maintain a "Always Green CI" aiming to improve the results of Kata Containers CI shown in GitHub.
+
+# Concepts
+
+In this section is discussed the main pillars that sustain the "Always Green CI" initiative.
```

Comment:
Change "section is discussed" to "section we discuss"

Reply:
Done